test/unit/full_text_search/searcher_test.rb

```diff
@@ -114,5 +114,73 @@ def test_order_registered_time_asc
                        .order(created_on: :asc)
       assert_equal(ordered_news, searched_news)
     end
+
+    def test_order_last_modified_time_desc
+      parameters = {
+        order_target: "last_modified_time",
+        order_type: "desc",
+        news: "1",
+        attachments: "0",
+        limit: "-1"
+      }
+      targets = search(parameters).records
+      searched_news = targets.collect(&:source_record)
+      ordered_news = @project
+                       .news
+                       .order(created_on: :desc)
```

Comment:
It seems that both of `registered_time` and `last_modified_time` use `created_on`.
It means that we can't detect a problem when we use `registered_at` for `order_target: "last_modified_time"`.
Can we use different data for better test?

Reply:
fix: [b3e0c78](https://github.com/clear-code/redmine_full_text_search/pull/121/commits/b3e0c78bff4817028ce855176b92bf4d21ccdeb7) Sure. I changed the test data from news to messages because both of registered_time and last_modified_time use different values like created_on or updated_on.

https://github.com/clear-code/redmine_full_text_search/blob/288b6676a7f4ec8b8ea46b148f1a23ad73884f22/lib/full_text_search/message_mapper.rb#L29-L30

---

fix: [2250376](https://github.com/clear-code/redmine_full_text_search/pull/121/commits/2250376f4b40efd95c71bda6d9c4e1bfcdae0cbc) Also, I fixed the order of generated test data for the score order test.